IDE/WICED-STUDIO/README

```diff
@@ -0,0 +1,44 @@
+The wolfSSL examples in the WICED SDK require a server or client application. wolfSSL
+provides an easy solution in the <wolfSSL root>/examples/ directory. Run the autogen
+script and configure the library with necessary functionality, see ./configure -h for a
+list of configure options. Finally, run make and make check to ensure the
+build was successfull. You can install wolfSSL on your system with make
+install if you wish. See www.wolfssl.com/docs/ for more information.
+
+Run ./examples/server/server -h for a list of server options or
+./examples/client/client -h for client options. If you would like
+to test the server or client, you may run them on localhost. Instructions on
+running the client or server can be found in the snip code examles. The snips are
+meant to be a starting off point for your applications.
+
+Download and install WICED Studio from the Cypress website.
+
+In the <wolfSSL_root>/IDE/WICED-STUDIO/ directory, you will find the library file
+user_settings.h which should be copied to <wolfSSL_root>. This is your wolfSSL
+configuration file. Please see www.wolfssl.com/docs/ for detailed instructions on
+how to use the configure options.
+
+Once you have installed the WICED software:
+    1. Create a directory called 'wolfssl_lib.' The path should be:
+                43xxx_Wi-Fi/WICED/security/BESL/wolfssl_lib.
+    3. Drop in the wolfssl_lib directory you just created.
```

Comment:
It's not clear to me where "wolfssl_lib" should be dropped, as it was just created in step #1.  Also, step 2 seems to be missing?

Reply:
Thanks